src/main/java/com/google/swarm/tokenization/beam/DLPDeidentifyText.java

```diff
@@ -293,9 +304,35 @@ public void processElement(ProcessContext c) throws IOException {
               .build();
       ContentItem contentItem = ContentItem.newBuilder().setTable(table).build();
       this.requestBuilder.setItem(contentItem);
-      DeidentifyContentResponse response =
-          dlpServiceClient.deidentifyContent(this.requestBuilder.build());
-      c.output(KV.of(fileName, response));
+
+      Sleeper sleeper = Sleeper.DEFAULT;
+      BackOff backoff = BUNDLE_WRITE_BACKOFF.backoff();
+
+      boolean retryApiFlag = true;
+      while(retryApiFlag){
+        try {
+          DeidentifyContentResponse response =
+                  dlpServiceClient.deidentifyContent(this.requestBuilder.build());
+          c.output(KV.of(fileName, response));
+          break;
+        }
+        catch(ResourceExhaustedException e) {
+            retryApiFlag = BackOffUtils.next(sleeper, backoff);
+            if(!retryApiFlag){
+                LOG.info("Max retries reached");
```

Comment:
Please use error level a message that looks like:
LOG.error("DLP API request quota exceeded. Retried {} times unsuccessfully. Some records were not de-identified. Exception {}", pipelineOptions.getDlpApiRetryCount(), e);
Please confirm that the error code, message make it to the logs correctly.